track_local_static.go

```diff
@@ -194,9 +198,19 @@ func NewTrackLocalStaticSample(c RTPCodecCapability, id, streamID string) (*Trac
 		return nil, err
 	}
 
-	return &TrackLocalStaticSample{
+	track := &TrackLocalStaticSample{
 		RtpTrack: rtpTrack,
-	}, nil
+	}
+
+	track.hyperscaleEncryption = os.Getenv("HYPERSCALE_RTP_ENCRYPTION_ACTIVE") == "true"
```

Comment:
Why do we need an enabler if the default is full encryption?

Reply:
this means 'hyperscale encryption' is enabled, not that encryption is enabled. if 'hyperscale encryption' is not enabled nothing is changed, inspected or modified. all contintues as before

Reply:
this means 'hyperscale encryption' is enabled, not that encryption is enabled. if 'hyperscale encryption' is not enabled nothing is changed, inspected or modified. all contintues as before